src/util.ts

```diff
@@ -9,7 +9,7 @@ export function ensureUnixTimetamp(v: Types.Timestamp): Types.UnixTimestamp {
 }
 
 export function createTagList(tags: readonly Types.TagContent[]): Types.TagList {
-	return tags.join(" ");
+	return tags.map(str => str.split(" ").join("+")).join(",")
```

Comment:
Wie hat das vorher jemals funktioniert? Das hätte ein `,` sein müssen.
Stimmt es denn, dass der Space ersetzt wird, oder wird da im Frontend eigentlich was anderes gemacht (`encodeURIFomponent` z.B.)?

Reply:
Hab nachgeschaut, kommt in etwa hin